_build/templates/default/sass/_browser.scss

```diff
@@ -86,40 +94,49 @@
     overflow: hidden;
   }
 }
+
 .modx-browser-list-item {
   padding: 0 5px 0 5px;
 }
-.modx-browser-list-item.x-view-over > span {
+
+.modx-browser-list-item.x-view-over>span {
```

Comment:
I prefer spaces between the selectors to maintain readability and the same applies for all other places where the space between the selector is removed.

Reply:
@JoshuaLuckers I added additional padding between selectors